<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ユリウス日の計算</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="monthBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>42</width>
      <height>23</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>12</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="dayBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>42</width>
      <height>23</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="hourBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>42</width>
      <height>23</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>23</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="minuteBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>42</width>
      <height>23</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="secondBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>42</width>
      <height>23</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="yearEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="calcButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>265</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>計算</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButto">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>265</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>クリア</string>
    </property>
   </widget>
   <widget class="QLabel" name="datelabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>22</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>年月日：</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>303</x>
      <y>22</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>時刻：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>265</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>閉じる</string>
    </property>
   </widget>
   <widget class="QComboBox" name="timeZoneTComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>日本標準時</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>世界時</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>461</width>
      <height>150</height>
     </rect>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <row/>
    <column>
     <property name="text">
      <string>年月日</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>時刻</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ユリウス日</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>修正ユリウス日</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignTrailing|AlignVCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignTrailing|AlignVCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="monthCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>268</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1日おきに1ヶ月間計算</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="nowButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>現在時刻</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zeroButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>０時</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
